tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java

```diff
@@ -702,4 +724,51 @@ public void theEventHasBeenAdded(String eventName) {
         JSONObject JSONWithEventAdded = addEventName(rawJSON, eventName);
         lambdaInput = wrapJSONObjectAsAnSQSMessage(JSONWithEventAdded);
     }
+
+    @Given("the SQS data file {string} is available for the {string} team")
+    public void the_sqs_data_file_is_available_for_the_team(String fileName, String account) throws IOException {
+        // Write code here that turns the phrase above into concrete actions
+        JSONObject rawJSON = new JSONObject(readJSONFile(fileName));
+        JSONObject enrichedJSON = addComponentIdAndTimestampFields(rawJSON, account);
+        lambdaInput = wrapJSONObjectAsAnSQSMessage(enrichedJSON);
+    }
+
+
+
+    public static void receiveSqsMessage(String CFN_SqsURL, int timeout) {
+        SqsClient sqs = SqsClient.builder().build();
+
+        // Get the receipt handle for the first message in the queue
+        ReceiveMessageRequest receiveRequest = ReceiveMessageRequest.builder()
+                .queueUrl(CFN_SqsURL)
+                .build();
+        Message message = sqs.receiveMessage(receiveRequest)
+                .messages()
+                .get(0);
+        sqsoutputtext = message.body();
+        DeleteMessageRequest deleteMessageRequest = DeleteMessageRequest.builder()
+                .queueUrl(CFN_SqsURL)
+                .receiptHandle(message.receiptHandle())
+                .build();
+        sqs.deleteMessage(deleteMessageRequest);
+        System.out.println("sqsoutput " +sqsoutputtext);
+    }
+
+
+    @Then("the SQS below should have a new event matching the respective {string} output file {string} in the {string} folder")
+    public void theSQSBelowShouldHaveANewEventMatchingTheRespectiveOutputFileInTheFolder(String arg0, String arg1, String arg2) throws JSONException, IOException {
+        receiveSqsMessage(ConfigurationReader.getSqsUrl(),30);
+
+        Path expectedJson = Path.of(new File("/Users/dennythampi/IdeaProjects/di-txma-audit/tests/event-processing-tests/src/test/resources/Test Data/AuthOIDC/Accounts_expected.json").getAbsolutePath());
```

Comment:
I think this would not resolve on the build pipeline when it's up and running again, perhaps try something like src/test/resources/Test Data/AuthOIDC/Accounts_expected.json

Reply:
Done